gitpod-io/gitpod
# Stream imagebuild logs directly from pods

## Description
Streams log URLs from the `imagebuild` workspace directly instead via `image-builder-mk3`, which made it cluster dependent.

Basically
 - old: `server` -> (cluster-local) `image-builder-mk3` -> `imgbuild` workspace
 - new: `server` -> `imgbuild` workspace

## Related Issue(s)

Fixes #7874

## How to test
- start an image build: https://gpl-7874-imagebuild-logs.staging.gitpod-dev.com/#https://github.com/geropl/roborally
- notice how the image build starts and logs are streamed shortly after. Once the build finished (failed), the state is reflected accordingly
- note: connecting to an already running image build does not seem to work atm because the `build(...)` call never returns from imagebuilder. I tried to find the root cause, but ultimately it's out-of-scope here.

## Release Notes

```release-note
none
```

## Documentation


- [x] /werft with-observability
